tasks.py

```diff
@@ -143,11 +106,7 @@ def package(ctx, function=None, upload_to_s3=False):
     """
     Package function(s) + deps into a zip file.
     """
-    if function is not None:
-        functions = [function]
-    else:
-        functions = FUNCTION_NAMES
-
+    functions = resolve_function_arg(function)
```

Comment:
I'm really disappointed in the invoke library for not having a way to handle manipulating incoming task arguments. also the `@task` decorator doesn't play nice with other decorators or I would have done it that way. oh well.